overlord/snapstate/snapmgr.go

```diff
@@ -174,7 +174,7 @@ type SnapState struct {
 	RefreshInhibitedTime *time.Time `json:"refresh-inhibited-time,omitempty"`
 
 	// LastRefreshTime records the time when the snap was last refreshed.
-	LastRefreshTime *time.Time `json:"last-refresh,omitempty"`
+	LastRefreshTime *time.Time `json:"last-refresh-time,omitempty"`
```

Comment:
So far this has only been set in the tests and not stored in state yet, changing the field name in JSON should be ok.

Reply:
Yep, it's for the upcoming refresh control feature.